go.mod

```diff
@@ -5,22 +5,23 @@ go 1.12
 require (
 	github.com/blang/semver v3.6.1+incompatible
 	github.com/go-ldap/ldap v3.0.3+incompatible // indirect
-	github.com/hashicorp/go-hclog v0.9.0 // indirect
+	github.com/hashicorp/go-hclog v0.9.2 // indirect
+	github.com/hashicorp/go-plugin v1.0.1 // indirect
 	github.com/lib/pq v1.1.1 // indirect
-	github.com/mattermost/go-i18n v1.10.0 // indirect
-	github.com/mattermost/mattermost-server v0.0.0-20190516103213-2d3e41783398
-	github.com/nicksnyder/go-i18n v1.10.0 // indirect
+	github.com/mattermost/go-i18n v1.11.0 // indirect
+	github.com/mattermost/mattermost-server v0.0.0-20190624195857-dbe330511eb4
```

Comment:
This can just be `v5.12.0+incompatible`

Reply:
Any idea how to get that effect with `go get`? Whenever I try `go get -u github.com/mattermost/mattermost-server` it gives me 5.11. Probably something to do with minimum version selection. So this is `go get -u github.com/mattermost/mattermost-server@release-5.12`. 

I did try `go get -u github.com/mattermost/mattermost-server@v5.12` but that just fails.

Reply:
I've run into this problem before. (https://github.com/mattermost/mattermost-plugin-autotranslate/pull/13#discussion_r294298918)

`go mod edit -require github.com/mattermost/mattermost-server@v5.12.0+incompatible` works. Yes, the `+incompatible is required.  (But don't ask me why)

Reply:
I thought we discussed this before :P 
Pushed an update after running your command. Some things about go modules still confuse me.